api/src/Api/Request/Data/TotalSocial/Import/ImportMetricListApiRequestData.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Api\Request\Data\TotalSocial\Import;
+
+use App\Api\Provider\MetricIntervalProvider;
+use App\Api\Request\Data\ApiRequestDataInterface;
+use App\Api\Services\VariableNameConverter;
+
+class ImportMetricListApiRequestData implements ApiRequestDataInterface {
+
+    public $ids = [70905, 70933];
+    public $metrics = 'offline.scoreVolume.value';
+    public $outputType = 'overtime';
+    public $from = 1442188800;
```

Comment:
$from = 1442188800 which represent September 14th, 2015 0:00:00?
why?

Reply:
it's value from my tests - I forgot to remove it